src/GitHub.App/ViewModels/PullRequestCreationViewModel.cs

```diff
@@ -186,6 +186,13 @@ public void Dispose()
             GC.SuppressFinalize(this);
         }
 
+       public override void Reset()
+        {
+            this.PRTitle = string.Empty;
+            this.Description = string.Empty;
+            TargetBranch = Branches.FirstOrDefault(b => b.Name == "master");
```

Comment:
At this point, I'm assuming I'll just need the `FirstOrDefault` `IBranch` and not have to search by name?

`Branches.FirstOrDefault(b => b == TargetBranch)`